descriptors/ModuleDescriptor-template.json

```diff
@@ -1078,6 +1078,30 @@
         }
       ]
     },
+    {
+      "id": "orders.bind",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["POST"],
+          "pathPattern": "/orders/bind",
```

Comment:
it is not clear what to bind from name, now it looks like we are binding orders together.
Propose to rename to bind-pieces for example